### PR TITLE
Make sure that result backend works out of box

### PR DESCRIPTION
**Pull request by bkyryliuk**
Helps with https://github.com/airbnb/superset/issues/1984#issuecomment-273442878

@mistercrunch ^^

### CODE REVIEW

File: superset/config.py [hunk 1]
```diff
@@ -222,7 +224,9 @@ class CeleryConfig(object):
 """
 CELERY_CONFIG = None
 SQL_CELERY_DB_FILE_PATH = os.path.join(DATA_DIR, 'celerydb.sqlite')
-SQL_CELERY_RESULTS_DB_FILE_PATH = os.path.join(DATA_DIR, 'celery_results.sqlite')
+SQL_CELERY_RESULTS_DB_FILE_PATH = os.path.join(
+    DATA_DIR, 'celery_results.sqlite')
+RESULTS_BACKEND = FileSystemCache('/tmp/sqllab_cache', default_timeout=60*24*7)
```
Comment (xrmx) [inline superset/config.py]: What about creating it inside DATA_DIR instead?

Comment (bkyryliuk) [inline superset/config.py]: @xrmx - good point, thanks !

**Comment by mistercrunch** [comment]
This isn't very explicit, I'm not sure if this is the right default. Administrators should probably be aware that this application will be caching/dumping results in the home directory...

**Comment by bkyryliuk** [comment]
@mistercrunch - what should be the right default ?

**Comment by astawiarski** [comment]
There shouldn't be a default. Similarly to the `BROKER_URL` it should be a well-documented parameter, but no defaults make sense and will only cause more trouble.

**Comment by mistercrunch** [comment]
I have a PR with better documentation coming

**Pull request closed without merging**